test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -412,9 +412,7 @@ def _run_dependency_check_test(image, ec2_connection):
         },
         "tensorflow": {
             "1.15": ["neuron"],
-            "2.5": ["neuron"],
-            "2.6": ["cpu", "gpu"],
-            "2.7": ["cpu", "gpu", "hpu"],
+            "2.5": ["cpu", "gpu", "neuron"],
```

Comment:
Please correct this by adding the `2.6` and `2.7` key-value pairs back.

Reply:
not sure on the origin of the change, added it back